tests/unit/s2n_aead_aes_test.c

```diff
@@ -272,18 +274,20 @@ int main(int argc, char **argv)
         conn->actual_protocol_version = S2N_TLS12;
         EXPECT_SUCCESS(bytes_written = s2n_record_write(conn, TLS_APPLICATION_DATA, &in));
 
-        const int overhead = 8   /* IV */
-            + 16; /* TAG */
-        if (i < max_fragment - overhead) {
+        if (i <= max_fragment) {
             EXPECT_EQUAL(bytes_written, i);
         } else {
-            EXPECT_EQUAL(bytes_written, max_fragment - overhead);
+            /* application data size of intended fragment size + 1 should only send max fragment */
+            EXPECT_EQUAL(bytes_written, max_fragment);
         }
 
         uint16_t predicted_length = bytes_written;
         predicted_length += conn->initial.cipher_suite->record_alg->cipher->io.aead.record_iv_size;
         predicted_length += conn->initial.cipher_suite->record_alg->cipher->io.aead.tag_size;
 
+        const int overhead = 8 /* IV */ + 16 /* TAG */;
```

Comment:
Seems to be duplicated a lot..

Reply:
I think this overhead is fine as it's only repeated twice